.buildkite/schedules/dependency-analysis.yml

```
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+common_params:
+  # Common plugin settings to use with the `plugins` key.
+  - &common_plugins
+    - automattic/a8c-ci-toolkit#2.15.0
```

```suggestion
    - automattic/a8c-ci-toolkit#3.4.2
```

suggestion as in other PR

---

same for the Login PR ([line](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/149/files#diff-65b833706f3d98174e386b58402db0fc43121edcc89ca03440b6ed9508adc2cbR7)), noted ([context](https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/148#discussion_r1666853762)).

---

FYI @wzieba that this is now done: #80